Deprecated `tidb_enable_table_partition`, `tidb_enable_list_partition` and `tidb_enable_global_index` (#18984)
pingcap/docs: partitioned-table.md

> **Note:**
> 

> > This rule can be ignored by using [global indexes](#global-indexes).

Reviewer: ```suggestion
> You can ignore this rule when using [global indexes](#global-indexes).
```

>

> > The global indexes feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

Reviewer: It seems that we should remove this warning about experimental global indexes? I see that this warning is removed in the PR https://github.com/pingcap/docs-cn/pull/18659 for docs in Chinese.

Reviewer 2: Removed already in this PR https://github.com/pingcap/docs/pull/18940/files

Reviewer: Got it.
